### PR TITLE
MC-13917: Update the api docs to combine hostname, path, and port fie…

### PR DESCRIPTION
**Pull request by BajajSid**
…lds into a single address field

#### Changes made
Update the API docs to sync up with the following changes: 
- [https://github.com/cloudops/cloudmc-stackpath-plugin/pull/198]

### CODE REVIEW

File: source/includes/stackpath/_origin_settings.md [hunk 1]
```diff
@@ -80,10 +74,7 @@ Attributes | &nbsp;
 `origin.password` <br/> *string* | Password to use when authenticating with the origin. 
 `origin.sslValidationEnabled` <br/> *boolean* | Specifies if SSL validation for the origin is enabled. 
 `origin.commonCertificateName` <br/> *string* | Common name to validate SSL origin requests against.
-`origin.path` <br/> *string* | The origin's HTTP request path.
-`origin.hostname` <br/> *string* | The origin's HTTP request hostname. Can be a valid IPv4 address or a valid domain name. 
-`origin.port` <br/> *int* | The origin's HTTP port. Must be one of [80, 8080, 443, 1935, 9091].
-`origin.securePort` <br/> *int* | The origin's HTTPS port. Must be one of [80, 8080, 443, 1935, 9091].
+`origin.address` <br/> *string* | The address of the primary origin that the CDN uses to pull content from. Can be a valid IPv4 address or a valid domain name. It may include a specific port and a precise path as well (e.g. 199.250.204.212:80/test).
```
Comment (patrickspensieri) [inline source/includes/stackpath/_origin_settings.md]: We should still document the list of valid ports

File: source/includes/stackpath/_origin_settings.md [hunk 2]
```diff
@@ -97,10 +88,7 @@ Attributes | &nbsp;
 `backupOrigin.password` <br/> *string* | Password to use when authenticating with the backup origin. 
 `backupOrigin.sslValidationEnabled` <br/> *boolean* | Specifies if SSL validation for the backup origin is enabled. 
 `backupOrigin.commonCertificateName` <br/> *string* | Common name to validate SSL origin requests against.
-`backupOrigin.path` <br/> *string* | The backup origin's HTTP request path.
-`backupOrigin.hostname` <br/> *string* | The backup origin's HTTP request hostname. Can be a valid IPv4 address or a valid domain name. 
-`backupOrigin.port` <br/> *int* | The backup origin's HTTP port. Must be one of [80, 8080, 443, 1935, 9091].
-`backupOrigin.securePort` <br/> *int* | The backup origin's HTTPS port. Must be one of [80, 8080, 443, 1935, 9091].
+`backupOrigin.address` <br/> *string* | The address of the secondary origin that the CDN uses to pull content from. Can be a valid IPv4 address or a valid domain name. It may include a specific port and a precise path as well (e.g. 199.250.204.212:80/test).
```
Comment (fyrnaga) [inline source/includes/stackpath/_origin_settings.md]: same comment as above

File: source/includes/stackpath/_origin_settings.md [hunk 3]
```diff
@@ -168,10 +156,7 @@ Attributes | &nbsp;
 `origin.password` <br/> *string* | Password to use when authenticating with the origin. 
 `origin.sslValidationEnabled` <br/> *boolean* | Specifies if SSL validation for the origin is enabled. 
 `origin.commonCertificateName` <br/> *string* | Common name to validate SSL origin requests against.
-`origin.path` <br/> *string* | The origin's HTTP request path.
-`origin.hostname` <br/> *string* | The origin's HTTP request hostname. Can be a valid IPv4 address or a valid domain name. 
-`origin.port` <br/> *int* | The origin's HTTP port. Must be one of [80, 8080, 443, 1935, 9091].
-`origin.securePort` <br/> *int* | The origin's HTTPS port. Must be one of [80, 8080, 443, 1935, 9091].
+`origin.address` <br/> *string* | The address of the primary origin that the CDN uses to pull content from. Can be a valid IPv4 address or a valid domain name. It may include a specific port and a precise path as well (e.g. 199.250.204.212:80/test).
```
Comment (fyrnaga) [inline source/includes/stackpath/_origin_settings.md]: Same comment as above

File: source/includes/stackpath/_origin_settings.md [hunk 4]
```diff
@@ -185,8 +170,5 @@ Attributes | &nbsp;
 `backupOrigin.password` <br/> *string* | Password to use when authenticating with the backup origin. 
 `backupOrigin.sslValidationEnabled` <br/> *boolean* | Specifies if SSL validation for the backup origin is enabled. 
 `backupOrigin.commonCertificateName` <br/> *string* | Common name to validate SSL origin requests against.
-`backupOrigin.path` <br/> *string* | The backup origin's HTTP request path.
-`backupOrigin.hostname` <br/> *string* | The backup origin's HTTP request hostname. Can be a valid IPv4 address or a valid domain name. 
-`backupOrigin.port` <br/> *int* | The backup origin's HTTP port. Must be one of [80, 8080, 443, 1935, 9091].
-`backupOrigin.securePort` <br/> *int* | The backup origin's HTTPS port. Must be one of [80, 8080, 443, 1935, 9091].
+`backupOrigin.address` <br/> *string* | The address of the secondary origin that the CDN uses to pull content from. Can be a valid IPv4 address or a valid domain name. It may include a specific port and a precise path as well (e.g. 199.250.204.212:80/test).
```
Comment (fyrnaga) [inline source/includes/stackpath/_origin_settings.md]: Same comment as above

File: source/includes/stackpath/_origin_settings.md (hunk 1 above)
Comment (BajajSid) [reply source/includes/stackpath/_origin_settings.md]: a408e2f

File: source/includes/stackpath/_origin_settings.md (hunk 2 above)
Comment (BajajSid) [reply source/includes/stackpath/_origin_settings.md]: a408e2f

File: source/includes/stackpath/_origin_settings.md (hunk 3 above)
Comment (BajajSid) [reply source/includes/stackpath/_origin_settings.md]: a408e2f

File: source/includes/stackpath/_origin_settings.md (hunk 4 above)
Comment (BajajSid) [reply source/includes/stackpath/_origin_settings.md]: a408e2f

**Review by patrickspensieri — APPROVED** [review]
Looks good!

**Review by fyrnaga — APPROVED** [review]
lgtm

**Pull request merged**